plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -8,13 +8,26 @@
 
 #include "fu-uefi-dbx-common.h"
 #include "fu-uefi-dbx-device.h"
+#ifdef WITH_UEFI_DBX_SNAPD_NOTIFIER
+#include "fu-uefi-dbx-snapd-notifier.h"
+#endif
 
 struct _FuUefiDbxDevice {
 	FuDevice parent_instance;
+#ifdef WITH_UEFI_DBX_SNAPD_NOTIFIER
+	FuUefiDbxSnapdNotifier *snapd_notifier;
+#endif
 };
 
 G_DEFINE_TYPE(FuUefiDbxDevice, fu_uefi_dbx_device, FU_TYPE_DEVICE)
 
+#ifdef WITH_UEFI_DBX_SNAPD_NOTIFIER
+static gboolean
+fu_uefi_dbx_device_maybe_notify_snapd_prepare(FuUefiDbxDevice *self, GBytes *data, GError **error);
+static gboolean
+fu_uefi_dbx_device_maybe_notify_snapd_cleanup(FuUefiDbxDevice *self, GError **error);
```

Comment:
could you place the functions before their first use in the file and avoid the prototypes?